evasquez26/angular4-startup
# S2 one 51 see course details

## Story / Task

[https://bixlabs.atlassian.net/browse/ONE-51](ONE-51)


## What this PR does?

Allows a user to check for course details


### Before 

We could not see course details

### After

We can see course details

### How to test / reproduce

- Create a course
- Go to created courses list
- Observe that clicking the view button takes you to the couse details view

### TODOs
- [ ] Rebase after merging #20 

### Related PRs
N/A

## Migrations

NO
